float32 x
float32 y
float32 diameter
int16 purpleProb
int16 redProb
int16 blueProb
int16 silverProb
int16 brassProb
bool searched
